Bang.Tests/DomainUnitTests/ServerMessageProcessorTests.cs

```diff
@@ -436,6 +436,65 @@ public void Change_weapon_message_removes_card_from_hand()
             Assert.DoesNotContain(card, player.PlayerHand);
         }
 
+        [Theory]
+        [MemberData(nameof(ReplenishCardsToCardsAmountMapping))]
+        public void Replenish_gand_card_message_returns_properly_cards_amount_in_message(BangGameCard card, int cardsShouldBeAdded)
+        {
+            var game = CreateAndStartGame();
+            var player = game.Players.First();
+            player.PlayerHand.Add(card);
```

Comment:
We have a lot of message processor tests that start with these lines. Maybe it makes sense to introduce a separate method for these three lines.

Reply:
Won't be fix